src/components/common/Extensions/ExtensionActionButton/ExtensionActionButton.tsx

```diff
@@ -39,6 +39,8 @@ const ExtensionActionButton = ({ extensionData }: Props) => {
     );
   };
 
+  const isSupportedColonyVersion = colony.version >= 5;
```

Comment:
There used to be a `ColonyVersion` enum exported from `colony-js` but it's not there anymore. Open to suggestions on replacing the hardcoded value

Reply:
I'd say have a constant 
``` js
const MIN_SUPPORTED_COLONY_VERSION = 5
const isSupportedColonyVersion = colony.version >= MIN_SUPPORTED_COLONY_VERSION;
```
or similar